csi/src/mount.rs

```diff
@@ -14,7 +14,6 @@ pub struct MountInfo {
 #[derive(Clone, Debug)]
 pub struct Fs {
```

Comment:
What about just putting strings in the supported list?

Reply:
Fixed by https://github.com/openebs/Mayastor/pull/263/commits/e24901e2e1dd4d5691e57b450068d981089f93a9